apps/aeutil/lib/list.ex

```diff
@@ -3,41 +3,45 @@ defmodule Aeutil.List do
   This module holds some additional utility functions for list
   """
 
-  @doc """
-  Merge two list in descending order
-  """
-  @spec merge_descending(list(), list()) :: list()
-  def merge_descending(list1, list2) do
-    merge(list1, list2, [])
+  @spec merge(list(), list()) :: list()
+  def merge(l1, l2) do
```

Comment:
Change to a more semantic correct name to avoid confusion with the ++ operator and the merge_descending function for instance: merge_sorted_lists_without_duplicates. Add docs saying that the resulting list is sorted and the inputs need to be sorted as well.

Reply:
When I gave it a bit more thought I decided to return the old functionality for few reasons:
1. The `peers` elements are not sorted to begin with, there is no functionality in chain of sync that does so;
2. When a `Chain` is initialized, we only have in the list of peers one peer_id, which is the peer from where the chain is coming;
3. When we do call `merge_chains`, what we do is take our already familiar chain and try matching it to an incoming one, if they do match, we merge them, so when we are merging the peers we are actually adding to our familiar chain just another peer;

For example if our familiar chain has already merged a few times and in the peers it has 3 peer_ids [100, 111, 1002], when we are merging to incoming chain what happens is this -> [100, 111, 1002] ++ [440]. Basically we are adding one element to the list, and we sort this list, because it's needed. For me this makes the merging function unneeded and only over-complicates the logic.